src/pages/CustomerRequisitionPage.js

```diff
@@ -76,8 +76,10 @@ export const CustomerRequisition = ({
   onSelectIndicator,
   onEditSuppliedQuantity,
   route,
+  onToggleColumnSet,
+  columnSet,
```

Comment:
So, to toggle columns have added a `columnSet` field to state, and a 'toggle'. Currently just using `'a'/'b'` as the `columnSets`